docs/docsite/rst/user_guide/sample_setup.rst

```diff
@@ -0,0 +1,286 @@
+.. _sample_setup:
+
+********************
+Sample Ansible setup
+********************
+
+You have learned about playbooks, inventory, roles, and variables. This section pulls all those elements together, outlining a sample setup for automating a web service. You can find more example playbooks illustrating these patterns in our `ansible-examples repository <https://github.com/ansible/ansible-examples>`_.  (NOTE: These may not use all of the features in the latest release, but are still an excellent reference!).
```

Comment:
Assuming this is page zero, which pages are we assuming they've already read?

Reply:
I'm not entirely sure at this stage. The last step in the User Guide rewrite will be revamping all the various TOCs within it. I'd prefer to treat every page as Page Zero, because most users will come from search engine results, but also offer a reasonable "Zero to Sixty for Ansible Engine" tutorial if you follow the TOC.